maubot.yaml

```diff
@@ -1,6 +1,6 @@
 maubot: 0.3.1
 id: me.jkhsjdhjs.maubot.webhook
-version: 0.3.0
+version: 0.4.0
```

Comment:
Can you remove this version bump from the PR? I'll create a separate, tagged version bump commit after this PR has been merged, which will also trigger a release.

Reply:
Sure, removed it. :+1: